pkg/asset/agent/agentconfig/agent_config.go

```diff
@@ -63,6 +64,12 @@ func (a *Asset) Load(f asset.FileFetcher) (bool, error) {
 		return false, errors.Wrapf(err, "failed to unmarshal %s", agentConfigFilename)
 	}
 
+	if config.Spec.RendezvousIP != "" {
+		common.AgentConfigAvaialble = true
+	} else {
+		return false, errors.New("missing Spec.RendezvousIP")
```

Comment:
We can't return an error here, the RendezvousIP is not _required_ (but you must have _either_ that _or_ at least one NMState config with a static IP).